contrib/sacrebleu/sacrebleu.py

```diff
@@ -658,7 +827,7 @@ def build_signature(args, numrefs):
     return sigstr
 
 
-def extract_ngrams(line, max_order=NGRAM_ORDER):
+def extract_ngrams(line, min_order=1, max_order=NGRAM_ORDER) -> Counter:
```

Comment:
you could generalize extract_ngrams to cover the character case as well. (either feed in a list of strings or a string (which is also indexable)).

Reply:
There is definitely some overlap here, but I gave up on combining them for diminishing returns. One problem is that a slice of a string is a string, but a slice of a list is a list, which can't work as a key for a Counter.